tests/unit/test_util.py

```diff
@@ -0,0 +1,27 @@
+import configparser
+
+import pytest
+
+from flask_container_scaffold.util import load_cfg
+
+
+def test_valid_cfg_file(mock_custom_only_extra_cfg):
+    """
+    GIVEN a valid cfg file
+    WHEN we try to load it
+    THEN we get back the expected python dict
+    """
+    expected = {'custom_section': {'Different_case': 'val',
+                                   'all_lower': 'something'}}
+    parsed = load_cfg(mock_custom_only_extra_cfg)
+    assert parsed == expected
+
+
+def test_invalid_cfg_file(mock_extra_settings_file):
+    """
+    GIVEN an invalid, Flask-style cfg file
+    WHEN we try to load it
+    THEN we get SOMETHING BAD
```

Comment:
Haha, maybe this should be the actual exception name?

Reply:
Haha, yeah, that was a placeholder when I didn't know the error yet :-)